### PR TITLE
Feature/ansible deployment

### PR DESCRIPTION
**Pull request by jordifranco**
Add an ansible way to setup the base raspberry pi
@ismferdev

### CODE REVIEW

**Review by ismferd — COMMENTED** [review]
is deployment/hosts.example correct?

**Review by ismferd — COMMENTED** [review]
you haven't got an inmutable infra... shouldn't you pin your packages?

File: deployment/roles/base/tasks/system.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+- name: Add user pi to group docker
+  user: name=pi groups=docker,pi,video append=yes shell=/bin/bash
+
+- name: Add pi to sudoers
+  lineinfile:
+    dest: /etc/sudoers
+    state: present
+    line: "pi ALL=(ALL) NOPASSWD: ALL"
+    insertafter: "EOF"
```
Comment (ismferd) [inline deployment/roles/base/tasks/system.yml]: possible blackhole?

File: deployment/roles/base/tasks/user.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+- name: Add a group pi
```
Comment (ismferd) [inline deployment/roles/base/tasks/user.yml:1]: Why do you need to do the same that system.yml?

File: deployment/setup.yml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+---
+- name: Setup Raspberry Pi Base system
+  hosts: raspberries
+  gather_facts: yes
+  remote_user: pi
```
Comment (ismferd) [inline deployment/setup.yml:5]: I want to connect with a non-root-user, don't I?

Comment (stealthizer) [reply deployment/setup.yml:5]: user pi is a non-root user with sudo privileges

**Pull request merged**